apps/asap-server/test/integration/fetch-users.integration-test.ts

```diff
@@ -1,35 +1,44 @@
+import Chance from 'chance';
+
 import { Role, UserResponse } from '@asap-hub/model';
 import { User } from '@asap-hub/squidex';
 import Users from '../../src/controllers/users';
 import { createUser } from '../helpers/users';
 
+const chance = new Chance();
 const users = new Users();
 
 describe('Users', () => {
   test('Should create and fetch a user', async () => {
+    const randomOrcid = chance.ssn();
+    const randomEmail = chance.email();
+    const randomName = chance.guid();
+
     const user: Partial<User> = {
       firstName: 'John',
-      lastName: 'Doe',
+      lastName: randomName,
       jobTitle: 'Project Manager',
-      orcid: '0000-0002-9079-593X',
+      orcid: randomOrcid,
       institution: 'Instituto Superior Tecnico',
-      email: 'john.doe@asap.science',
+      email: randomEmail,
       role: 'Grantee' as Role,
       degree: 'MPH',
     };
 
     await createUser(user);
 
-    const result = await users.fetch({});
+    const result = await users.fetch({
```

Comment:
or we could just fetch the user by ID? although then we would have to specify the ID on creation so maybe some other time